rust/src/command/internal.rs

```diff
@@ -41,12 +52,16 @@ pub enum InternalCommandWithData {
         amount: u64,
         state_hash: BlockHash,
         kind: InternalCommandKind,
+        date_time: i64,
```

Comment:
```suggestion
        date_time: u64,
```
??

Reply:
it should be but the DateTime uses i64. I do it for convenience rather than doing try_into and potentially being lossy